CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -8,7 +8,8 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ### etcd server
 - Fix [Remove memberID from data corrupt alarm](https://github.com/etcd-io/etcd/pull/14852).
-- Fix [non mutating requests pass through quotaKVServer when NOSPACE](https://github.com/etcd-io/etcd/pull/14884)
+- Fix [non mutating requests pass through quotaKVServer when NOSPACE](https://github.com/etcd-io/etcd/pull/14884).
```

Comment:
Could you, please, add word 'Allow' it is: 'Fix: [Allow non mutating...].

Without this I was not sure whether it's good that we let them go through, or it's unexpected and we 'disallowed' for some reason.

Reply:
I did not change this item, I just added a dot at the end. 

Updated anyway. thx